lib/pxapi.js

```diff
@@ -60,8 +62,9 @@ function buildRequestData(ctx, config) {
     };
 
     if (ctx.graphqlData) {
```

Comment:
This should be added to spec - multiple operation logic

Reply:
it is in spec, fixed to fit